distribution/graal/dist-material/bin/exportEnv.sh

```diff
@@ -19,4 +19,5 @@
 
 export SW_CONFIG_PATHS=${OAP_HOME}/config
 export SW_ENVOY_METRIC=default-graalvm
-export SW_LOG_ANALYZER=default-graalvm
\ No newline at end of file
+export SW_LOG_ANALYZER=default-graalvm
+export SW_OAL_ENGINE_DEBUG=true
```

Comment:
Is this file controlling compiling stage or runtime stage? I feel these are mixed somehow.

Reply:
Should be runtime.
My fault, we do not need to set SW_OAL_ENGINE_DEBUG